XCHammer.yaml

```diff
@@ -1,5 +1,5 @@
 targets:
-    - ":xchammer"
+    - "//:xchammer"
 
 projects:
     "xchammer-bazel":
```

Comment:
lets just call this xchammer now

Reply:
@rahul-malik as long as we have the SPM target around, it will conflict